File: 参考/comp9041-454/9041-labonserver/test11/median_number.pl
#!/usr/bin/perl -w
@list = ();
foreach $ele (@ARGV){
    push @list, $ele;
}
@list = sort{$a <=> $b} @list;
$list_len = @list;
$index = ($list_len-1)/2;
print "$list[$index]\n";

File: 参考/comp9041-Q/cs9041/test7/median_number.py
#!/usr/bin/python3
import sys
nums = sorted(sys.argv[1:],key = lambda x:int(x))
print(nums[len(nums)//2])





















# import math,sys
# l = sys.argv[1:]
# for i in range(len(l)):
#     l[i] = int(l[i])
# l.sort();
# print(l[len(l)//2])
